plugin-flex-ts-template-v2/src/feature-library/dispositions/custom-components/DispositionTab/DispositionTab.tsx

```diff
@@ -78,6 +122,10 @@ const DispositionTab = (props: OwnProps) => {
     updateStoreDebounced();
   }, [notes]);
 
+  useEffect(() => {
```

Comment:
I believe you can combine the above `useEffect` with this one. Just have both `customAttributes` and `notes` as a `useEffect` dependency.